app_list.js

```diff
@@ -33,7 +33,7 @@ app.controller("app_list", function($scope, $http) {
     });
        
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/app_list.js
+++ b/app_list.js
@@ -31,7 +31,7 @@
                 source: searchEngine
         });        
     });
-       
+
 
     $scope.categoryFilter = function (event) {
         item = event.target.id;
```